### PR TITLE
add korea south and korea central to acs preview regions

### PR DESCRIPTION
**Pull request by rjtsdl**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [x] Each command and parameter has a meaningful description.
- [x] Each new command has a test.

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

**Comment by rjtsdl** [comment]
@troydai TAL :)

File: src/command_modules/azure-cli-acs/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+unreleased
+++++++++++
+* add korea south and korea central to preview regions
```
Comment (mboersma) [inline src/command_modules/azure-cli-acs/HISTORY.rst]: @rjtsdl I think this note should go under `2.0.22` instead. Travis CI said:
`We no longer require 'unreleased' in headings. Use the appropriate version number instead.`

**Comment by mboersma** [comment]
I'll make sure this change gets into the KnackConversion refactoring for aks. Almost done!

**Comment by mboersma** [comment]
An equivalent change to this was merged in #5036. My understanding is that `KnackConversion` branch will be merged to `dev` or otherwise used for the next CLI release, so this PR isn't strictly necessary now.

**Comment by rjtsdl** [comment]
If that is true, we can close this PR. @mboersma

**Comment by derekbekoe** [comment]
@rjtsdl At the time, @mboersma was correct but the next release will happen from the dev branch so please fix this PR and we can merge.

**Comment by rjtsdl** [comment]
@derekbekoe thx, just fixed. Please merge when Travis pass

**Comment by mboersma** [comment]
@rjtsdl I think the tests failed because it wants the change under the existing `2.0.21` header in the change history. Honestly not sure what the mechanism here is, but empirical testing through previous PRs showed that to be the case.

**Comment by rjtsdl** [comment]
@mboersma, i was getting error when i put it under 2.0.21. I also get error.

**Comment by derekbekoe** [comment]
> The topmost version in /home/travis/build/Azure/azure-cli/src/command_modules/azure-cli-acs/HISTORY.rst does not match version 2.0.21 defined in setup.py.

Since the version on PyPI is 2.0.21, modify setup.py to 2.0.22 then move the change in HISTORY to 2.0.22

**Comment by derekbekoe** [comment]
CI is failing for unrelated reason. Some prev. commits should have updated the version numbers of their modules but somehow slipped through.
Will merge then fix in another PR.

**Comment by rjtsdl** [comment]
Yes, sounds good. I was about to ask you merge :)

**Pull request merged**